docs/buildah-from.md

```diff
@@ -47,13 +41,13 @@ If an image needs to be pulled from the registry, suppress progress output.
 
 ## EXAMPLE
 
-buildah from imagename --pull --transport "docker://myregistry.example.com/"
+buildah from imagename --pull
 
 buildah from docker://myregistry.example.com/imagename --pull
 
 buildah from imagename --signature-policy /etc/containers/policy.json
 
-buildah from imagename --pull-always --transport "docker://myregistry.example.com/" --name "mycontainer"
+buildah from docker://myregistry.example.com/imagename --pull-always --name "mycontainer"
```

Comment:
It would be awesome to point to skopeo's manpage wrt different transports.

Reply:
> I could also cut and paste it into the other man pages.

Eventually there probably should be a centralized man page, something like `container-transports(5)`, which everything could refer to; as we add new transports (or worse, change the semantics of existing transports, as is now happening with the `oci` image tag→reference name), there would be only one page to update instead of dozens; and if that man page lived in containers/image, it could be updated by the PR which changes the implementation.  Only… no such page exists yet I’m afraid.